src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,20 @@
-import React from 'react';
+// import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[],
+};
+
+export const CommentList = ({ comments }: Props) => {
+  if (!comments.length) {
+    return <b data-cy="NoCommentsMessage">No comments yet</b>;
```

Comment:
Here <hr ></> is missing, look at original html.

Reply:
Yes, that's true. You can add it to top of `PostInfo` or you have to add it here